src/main/java/hudson/plugins/timestamper/action/TimestampsActionOutput.java

```diff
@@ -95,7 +95,7 @@ static BufferedReader open(
       for (Function<Timestamp, String> format : query.timestampFormats) {
         parts.add(format.apply(currentTimestamp));
       }
-      String result = Joiner.on(' ').join(parts) + "\n";
+      String result = Util.join(parts, " ") + "\n";
```

Comment:
is this not just
```suggestion
      String result = String.join(" ", parts) + "\n";
```

I would have thought that `Util.join` was Deprecated (or should be soon)?

Reply:
Done